scripts/package_building/check_latest_tags.sh

```diff
@@ -20,7 +20,11 @@ function get_latest_versions {
 
     pushd "./${repo_name}"
     # Get latest 2 versions
-    ver="$(git tag | sort --version-sort --reverse | grep -E "^v[0-9]{1,3}\.[0-9]{1,3}$" -m2)"
+    if [[ $kernel_tree =~ "dpdk" ]];then
```

Comment:
This script is designed to give you the two valid tags. I think the decision to reject the tags/versions can be done in linux_pipeline/Jenkinsfile_dpdk_pipeline script.